.github/workflows/build.yml

```diff
@@ -23,10 +23,9 @@ jobs:
       - uses: actions/checkout@v2
       - run: npm install
       - run: npm run coverage
-      - name: Upload coverage to Coveralls
-        uses: coverallsapp/github-action@master
+        uses: codecov/codecov-action@v1
```

Comment:
```suggestion
      - uses: codecov/codecov-action@v1
```

Reply:
Oh. Will integrate locally and push again.

Reply:
On a second thought I have to say though that I completely do not understand when to use `-` and when not in these actions config `yml` files. 😜

Reply:
haha yeah that diff was a bit confusing since the `- name:` line was removed, it would have been easier to parse if i just renamed it but i guess i decided it wasn't necessary in the devp2p PR haha. the name is just an aesthetic that titles the run in the actions UI, but pretty self-explanatory as a codecov action.

the `-` signifies a new command then everything under it as properties, but the indentation is strict and always tricky.